scripts/github_pr/github_pr_cloud.yaml

```diff
@@ -46,6 +47,27 @@ template:
             parameters:
               status: error
               message: Owner of the source repo for this PR lacks permission
+      - type: FileMatch
+        config:
+          pass_through: all
+          paths:
+            -  !ruby/regexp '/scripts\/jenkins\/ardana\//'
+            -  !ruby/regexp '/jenkins\/ci.suse.de\/pipelines\/openstack-ardana.*\.Jenkinsfile$/'
+        whitelist_handler:
+          - type: JenkinsJobTriggerArdana
+            parameters:
+              detail_logging: true
+              job_name: openstack-ardana
+              job_cmd: "../jenkins/jenkins-job-trigger"
+              job_parameters:
+                <<: *ardana_automation_parameters
+        blacklist_handler:
+          - type: SetStatus
+            config:
+              context: suse/ardana
```

Comment:
If we follow the context-string model you have described, then how can we make the ardana job required for changes that affect ardana? If one job sets `suse/mkcloud` and another sets `suse/mkcloud/ardana`, then can we make both contexts required?

I was attempting to put them side-by-side so that they could both be required (and set them successful when the jobs are skipped) and so that it would be obvious to the developer which jobs to look at after a failure.